File: 861-flipping-an-image/flipping-an-image.py
class Solution:
    def flipAndInvertImage(self, image: List[List[int]]) -> List[List[int]]:
        
        for i,row in enumerate(image):
            image[i] = image[i][::-1]
        
        n = len(image)
        m = len(image[0])
        ans = [[0]*m for _ in range(n)]

        for i in range(n):
            for j in range(m):
                if image[i][j] == 0:
                    ans[i][j] = 1
                else:
                    ans[i][j] = 0

        return ans 

File: 861-flipping-an-image/flipping-an-image.cpp
class Solution {
public:
    vector<vector<int>> flipAndInvertImage(vector<vector<int>>& image) {
        int n = image.size();
        int m = image[0].size();

        for(int i=0;i<n;i++){
            reverse(image[i].begin(),image[i].end());
        }

        for(int i=0;i<n;i++){
            for(int j=0;j<m;j++){
                image[i][j] = image[i][j] == 0 ? 1 : 0;
            }
        }

        return image;
    }
};